sdks/workflow-browser-sdk/src/lib/tests/injections.test.ts

```diff
@@ -28,39 +29,4 @@ describe('injections', () => {
       actions: ['USER_NEXT_STEP'],
     });
   });
-
-  it('should inject plugins', () => {
```

Comment:
It looks like a valid test, im not sure we want to remove it instead of fixing it